src/python/test/sugar/test_objects.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import print_function, unicode_literals, division, absolute_import
```

Comment:
I'll take a closer look at this module tomorrow and make a PR back.

AFAIK In `dx-toolkit`, the test don't assume any prior states and when they execute they'll lack access to projects or objects. I see that we use:

* [Temporary projects](https://github.com/dnanexus/dx-toolkit/blob/master/src/python/test/dxpy_testutil.py#L143) when dealing with setting up temp projects.
* [DXTestCaseBuildWorkflows](https://github.com/dnanexus/dx-toolkit/blob/master/src/python/test/dxpy_testutil.py#L463) to handle building test workflows when needed.

We should check with`dx-toolkit` maintainers with how they want these types of temporary resources handled.